spec/support/factory_bot.rb

```diff
@@ -5,6 +5,7 @@
     sequence(:email) {|n| "user#{n}@example.com" }
     password 'password'
     password_confirmation 'password'
+    username 'username_1'
```

Comment:
if you require a unique username you will have to use a `sequence` invocation like we've done with the email above ^